manager/assets/modext/widgets/resource/modx.window.resource.js

```diff
@@ -1,7 +1,69 @@
-MODx.window.CreateResource = function(config) {
-    config = config || {};
-    this.ident = config.ident || 'tplpick'+Ext.id();
-    var id = this.ident;
+MODx.window.CreateResource = function(config = {}) {
+    this.ident = config.ident || `tplpick${Ext.id()}`;
+    const id = this.ident,
+          requireAlias = parseInt(MODx.config.friendly_urls, 10) && !parseInt(MODx.config.automatic_alias, 10),
+          aliasLength = parseInt(MODx.config.friendly_alias_max_length, 10) || 0,
+          resourceDetail = [
+              {
+                  columnWidth: requireAlias ? 0.33 : 0.5,
```

Comment:
Widths don't actually have to add up to 1 specifically. ExtJS will total all widths, divide by number of columns, and size accordingly. So this conditional isn't technically needed. The conditional also doesn't hurt, so no need to go back and remove it, but just FYI.

Reply:
Will take a look at that for future consideration.